src/main/java/appeng/integration/modules/CLApi.java

```diff
@@ -20,19 +20,26 @@
 
 
 import appeng.api.util.AEColor;
+import appeng.helpers.Reflected;
 import appeng.integration.BaseModule;
+import appeng.integration.IntegrationHelper;
 import appeng.integration.abstraction.ICLApi;
 
 
 public class CLApi extends BaseModule implements ICLApi
 {
-
+	@Reflected
 	public static CLApi instance;
 
+	@Reflected
+	public CLApi()
+	{
+		IntegrationHelper.testClassExistence( this, coloredlightscore.src.api.CLApi.class );
+	}
+
 	@Override
 	public void init() throws Throwable
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1186)